lib/src/logging/logging.cc

```diff
@@ -71,8 +69,12 @@ namespace facter { namespace logging {
                 // Since logging is busted, we raise an exception that
                 // signals to the consumer that a special action must
                 // be taken to alert the user.
-                throw locale_error(e.what());
+                throw locale_error(string("could not initialize logging, even with locale variables reset to LANG=C LC_ALL=C: ")
```

Comment:
This loses any potential information that e.what() may have. Are we sure there's nothing potentially useful there?